### ISSUE TITLE
Create template for Green MAF

### ISSUE THREAD

**Issue by gowebUSA** [gowebUSA/MSSA-Project#20]
As a MC, I want to print the MAF using Green MAF layout for easy filing of paper MAFs.

- [x] Use old excel file and save as web html file and modify all rows and columns.
- [x] Adjust to make sure it fits on a single sheet of paper.

Accepted Criteria: When able to print on one page and all fields are filled out on proper place.

**Comment by gowebUSA** [comment]
Already modifying the excel web html file sheets. 
   * Started inputting labels on blank cells to create a template and for easy manipulation of cell size and fill.
   * White MAF.htm has the XML format. I am working on Sheet002 so I had to replace that.
   * I am manipulating the Sheet002 file inside White MAF_files. Labeling them for easy identification of cells later when I am ready to implement codes inside each cell.

**Comment by gowebUSA** [comment]
Get with Eli for login functionality.

**Comment by gowebUSA** [comment]
I'm modifying this file to make changes to the white MAF. https://github.com/gowebUSA/MSSA-Project/blob/master/files/White%20MAF_files/sheet002.htm

**Comment by gowebUSA** [comment]
All mso css templates works. Everytime I launch the DETAILS link on the Workload tab, at first, the form doesn't look like it's calibrated. I checked the viewsource code and inspection mode and found out that stylesheets.css were empty. So I re-published the stylesheet to Azure and it worked. I also moved the Aircraft Details body below the form so when we print, it can capture the whole form on one page.
